### PR TITLE
HW (Masatoshi Nishiguchi)

### PR DESCRIPTION
**Pull request by mnishiguchi**
http://mnishiguchi.github.io/atm/

``` yaml
comfort:  5
complete: 5
```

### CODE REVIEW

File: atm.js [hunk 1]
```diff
@@ -1,3 +1,219 @@
...
+/**
+ * A class that represents user account.
+ */
+function Account() {
+
+  // State
+  this.balances = {
+    checking: 0,
+    savings: 0
+  };
```
Comment (jshawl) [inline atm.js:26]: Nice job using OOP!!

File: atm.js [hunk 2]
```diff
@@ -1,3 +1,219 @@
...
+
+  /**
+   * Processes the deposit for the specified account type and amount.
+   * @param  {[String]} type   either "checking" or "savings"
+   * @param  {[String]} amount a numerical string
+   * @return true if the transaction was successful, else false.
```
Comment (jshawl) [inline atm.js:32]: where did you learn this commenting convention? helpful!

File: atm.js [hunk 3]
```diff
@@ -1,3 +1,219 @@
...
+   */
+  this.deposit = function( type, amount ) {
+
+    // Reject if input is invalid.
+    if ( ! this.isValidAmountFormat( amount ) ) {
+      return false;
```
Comment (jshawl) [inline atm.js:38]: maybe tell the user about it?

File: index.html [hunk 4]
```diff
@@ -27,5 +27,8 @@
       <input class="deposit" type="button" value="Deposit" />
       <input class="withdraw" type="button" value="Withdraw" />
     </div>
+
+    <!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"|script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script> -->
```
Comment (jshawl) [inline index.html:31]: 👍

**Comment by jshawl** [comment]
Holy smokes awesome job on this assignment!!! 🌟

File: atm.js (hunk 2 above)
Comment (mnishiguchi) [inline atm.js:32]: At first, I learned Java's convention when I was learning Android last year. https://en.wikipedia.org/wiki/Javadoc
I am trying to make sure that everything is readable for my future self.

File: atm.js (hunk 1 above)
Comment (mnishiguchi) [inline atm.js:26]: Thanks! I am glad you liked it.

**Pull request closed without merging**